src/api-specification/contribution-controller/retrieving-top-contributions.md

```diff
@@ -17,7 +17,8 @@
 | `ft` | `array[string]` | The type of the fraud event. <br> Could be one of the following: <ol><li>`Wangiri`</li><li>`IRSF`</li><li>`StolenDevice`</li><li>`IPFraud`</li><li>`SMSA2P`</li></ol> |
 | `org` | `array[string]` | The country of origination of the fraud event. Set as a two-digit country code (Alpha-2, ISO 3166; e.g., US, GA) |
 | `self-only` | `boolean` | A boolean that defines whether the response should only contain contributions submitted by the requesting peer. <br> The `self-only` parameter is set to `false` by default. |
-| `fetch-mode` | `string` | The parameter that filters the response entries based on whether the requesting user has already seen them. <br> Could be one of the following: <ol><li>`NEW` — includes only the unseen by the requesting user entries.</li></ol><li>OLD — includes only the already seen by the requesting user entries.</li><li>`DEFAULT` — includes all of the entries.</li> <br> The `fetch-mode` parameter is set to `DEFAULT` by default. |
+| `fetch-mode` | `string` | The parameter that filters the response entries based on whether the requesting user has already seen them. <br> Could be one of the following: <ol><li>`NEW` — includes only the unseen by the requesting user entries.</li><li>`OLD` — includes only the already seen by the requesting user entries.</li><li>`DEFAULT` — includes all of the entries.</li></ol> <br> The `fetch-mode` parameter is set to `DEFAULT` by default. |
+| `confidence-score` | `boolean` | A boolean that defines whether the price of the retrieved contributions is affected by the [confidence index](../../overview/tokenomics.md#confidence-index). <br> The `confidence-score` parameter is set to `false` by default. |
```

Comment:
it seems we miss a description of the fact that contributions with a confidence index assigned are expected to be more expensive that the ones without an index
`price affected` does not give enough clarity imo

Reply:
Added to the "Tokenomics" and "Retrieving top contributions" topics